python/src/trezorlib/cli/cardano.py

```diff
@@ -63,9 +63,33 @@ def sign_tx(client, file, protocol_magic, network_id, testnet):
     outputs = [cardano.create_output(output) for output in transaction["outputs"]]
     fee = transaction["fee"]
     ttl = transaction["ttl"]
+    certificates = []
```

Comment:
```
certificates = [
    cardano.create_certificate(certificate)
    for certificate in transaction.get("certificates", ())
￼]
```
dtto for withdrawals

Reply:
Fixed [here](https://github.com/trezor/trezor-firmware/pull/1145/commits/3271673191120168e00097443f841a25a0711583).